### DISCUSSION TITLE
I met a problem when trying to recognize one-digit numbers and %

### DISCUSSION THREAD

**Discussion by NataliaAssange** [JaidedAI/EasyOCR#811] [Q&A]
I'm using this code to recognize a chart like this
![image11](https://user-images.githubusercontent.com/36265870/183017923-83adfa08-23a9-4980-84a8-88001aaea36e.png)
but I have 2 problems
1. the one-digit numbers (like sales in single month) can't be recognized
2. the % is often recognized as 6,p, or similar characters
my code is `reader.readtext(picpath, min_size=0,detail=0,width_ths=0.7)`
what should I do?
Thank you!

**Comment by tam0w** [comment]
for columns where you have one digit number its better to ocr each element in the that column one by one and then lower threshold for each ocr operation to 0.

if you want it to return a value when you only give the number '0', make sure to lower the threshold, text_threshold values to 0 in the readtext parameter for the reader class. CRAFT has issues detecting characters when only single characters are there, so if you lower the threshold for character detection, it will detect it.
